draft-gpew-priv-ppm.md

```diff
@@ -771,13 +765,10 @@ and constructs its response to the aggregate request.
 
 #### Leader State
 
-The leader is required to issue aggregate requests in order, but reports are
-likely to arrive out-of-order. The leader SHOULD store reports for a time
-period proportional to the batch window before including them in an aggregate
-request. Failure to do so may result in out-of-order reports being dropped by
-the helper. The leader MUST NOT accept reports whose timestamps are in the
-future. Implementors MAY provide for some small leeway, usually no more than a
-few minutes, to account for clock skew.
+The leader is required to buffer reports while waiting to aggregate them. The
+leader MUST NOT accept reports whose timestamps are too far in the future.
+Implementors MAY provide for some small leeway, usually no more than a few
+minutes, to account for clock skew.
```

Comment:
The point isn't that it is too far into the future, it is that you either track nonces, or you reject the input.

Reply:
Fair enough, but this text isn't new to this PR. What's changed here is the need to re-order reports.